pkg/resource/k8s-events-config/grafana-agent/events-logger-config.grafanaagent.yaml.template

```diff
@@ -0,0 +1,15 @@
+# This file was generated by logging-operator.
+# It configures the Grafana-agent to be used as events logger.
+# - configMap is generated from events-logger.grafanaagent.template and passed as a string
+#   here and will be created by Grafana-agent's chart.
+# - Grafana-agent runs as a deployment, with only 1 replica.
+grafana-agent:
+  agent:
+    configMap:
+      content: |-
+{{ .GrafanaAgentInnerConfig | indent 8 }}
+  controller:
+    replicas: {{ .Replicas }}
+    type: {{ .Type }}
+  crds:
+    create: {{ .Create }}
```

Comment:
```suggestion
    create: false

```

Reply:
We used a const for reasons in the past so probably those can go away

Reply:
Ok done.